checkDataQuality.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/python3
+
+import sys
+import os
+import getopt
+import asyncio
+from aiohttp import ClientSession
+import csv
+import json
+from datetime import datetime
+import random
+import numpy
+
+OUTPUT_DELIMITER = ';'
+STATEMENT_COUNT_URL = 'https://www.wikidata.org/w/api.php?format=json&action=query&prop=pageprops|revisions&ppprop=wb-claims&rvprop=ids'
+SITELINK_COUNT_URL = 'https://www.wikidata.org/w/api.php?format=json&action=wbgetentities&props=sitelinks'
+CONSTRAINT_CHECK_URL = 'https://www.wikidata.org/w/api.php?format=json&action=wbcheckconstraints'
+ORES_URL = 'https://ores.wikimedia.org/v3/scores/wikidatawiki?revids='
+
+batchSize = 10
+
+def usage(exitCode = False):
+    print('checkConstraints.py -i <inputfile> | -r <number of items> [-o <outputfile> -b <batch-size>]')
```

Comment:
File name is updated to `checkQualityData.py`

Reply:
fixed in [0df29b6](https://github.com/wmde/wikidata-constraints-violation-checker/pull/29/commits/0df29b6bb156a025a5e5267c923b80f251d8ca07)